geode-management/src/main/java/org/apache/geode/management/configuration/AbstractConfiguration.java

```diff
@@ -47,7 +47,7 @@
 @Experimental
 public abstract class AbstractConfiguration<R extends RuntimeInfo>
     implements Identifiable<String>, JsonSerializable {
-
+  private static final long serialVersionUID = -6612840641128145954L;
```

Comment:
Optional: all JDK classes put serialVersionUID at the bottom of the class. I usually do this to try and be consistent but obviously doesn't matter much.